package.json

```diff
@@ -44,16 +44,18 @@
     "emotion-theming": "9.2.9",
     "focus-trap-react": "4.0.1",
     "memoize-one": "4.0.2",
-    "react": "16.5.2",
     "react-click-outside": "3.0.1",
     "react-delegate-component": "1.0.0",
-    "react-dom": "16.5.2",
     "react-emotion": "9.2.12",
     "react-toggled": "1.2.7",
     "react-transition-group": "2.5.0",
     "react-virtualized": "9.20.1",
     "semantic-release": "15.9.6"
   },
+  "peerDependencies": {
+    "react": ">=15.4.1 <=16.7.0",
```

Comment:
`16.7.0` will still work for `16.7.0-alpha.0`, right?

Reply:
Could we just say <= 16.7?

Reply:
I think 16.7.0 is released anyway

Reply:
I'm not sure Hooks are available outside of non-alpha versions?

Reply:
They're not listed in the 16.7.0 changelog

Reply:
I assume npm considers `16.7.0-alpha.0` as matching `>=15.4.1 <=16.7.0`. If not, we could do `>=15.4.1 <=16.7.0 || 16.7.0-alpha.0`

Reply:
I'll change it